webpack/base.config.ts

```diff
@@ -9,7 +8,8 @@ const configuration: Configuration = {
     globalObject: 'this',
     path: resolve('./lib'),
     filename: 'index.js',
-    libraryTarget: 'umd'
+    libraryTarget: 'umd',
```

Comment:
Ser forresten fra webpack-dok: Please use [output.library.type](https://webpack.js.org/configuration/output/#outputlibrarytype) instead as we might drop support for output.libraryTarget in the future.